### PR TITLE
fix: review feedback for cards

### PR DESCRIPTION
**Pull request by davidmenendez**
Contributes to https://github.com/carbon-design-system/ibm-cloud-cognitive/issues/1086

### CODE REVIEW

**Review by matthewgallo — COMMENTED** [review]
image
The mediaRatio console error is gone now, however there is now a warning

File: packages/cloud-cognitive/src/global/js/package-settings.js [hunk 1]
```diff
@@ -13,10 +13,12 @@ const defaults = {
     // reviewed and released components:
     AboutModal: true,
     APIKeyModal: true,
+    Card: true,
```
Comment (dcwarwick) [inline packages/cloud-cognitive/src/global/js/package-settings.js:16]: I don't think Card should be listed here -- it's not a component our end-users will use directly. It's internal, like TearsheetShell.

```suggestion
```

**Review by dcwarwick — DISMISSED** [review]

**Comment by matthewgallo** [comment]
It looks like the snapshot test is failing, I just had this happen on one of my PRs. If you run `yarn` on your local branch to update the dependencies, and then run `yarn jest styles -u` and commit the new snapshot, you should be good.

**Review by matthewgallo — APPROVED** [review]
mediaRatio warning in storybook is still there, but everything else looks good!

File: packages/cloud-cognitive/src/global/js/package-settings.js (hunk 1 above)
Comment (matthewgallo) [reply packages/cloud-cognitive/src/global/js/package-settings.js]: This has been addressed in one of @davidmenendez's latest commits

**Pull request merged**